website/pages/docs/job-specification/hcl2.mdx

```diff
@@ -0,0 +1,95 @@
+---
+layout: docs
+page_title: HCL2
+sidebar_title: HCL2 <sup>Beta</sup>
+description: Learn about the Job specification HCL2 used to submit jobs to Nomad.
+---
+
+# HCL2 <sup>Beta</sup>
+
+~> **NOTE:** This page is about Nomad 1.0 Beta. Details are subject to change before final release.
+
+Nomad 1.0 adopts
+[HCL2](https://github.com/hashicorp/hcl2/blob/master/README.md), the second
+generation of HashiCorp Configuration Language. HCL2 extends the HCL language by
+adding expressions and input variables support that improves Job Spec
+reusability and readability. Also, the new HCL2 parser improves the error
+messages for invalid jobs.
+
+## Expressions
+
+Nomad 1.0 supports expressions and built-in functions, allowing for a more concise job spec. HCL2 allows use of the following expressions:
+
+- value literals: e.g. `42`, `"hello"`, `true`
+- arithmetic expression: `5*100`
+- for-each expressions: `[for s in [3, 4, 5]: s+1]` (which evaluates to `[4, 5, 6]`)
+- invocation of function calls: `upper("FOO")`.
+
+For a full list of available functions, see [the function reference](https://www.packer.io/docs/from-1.5/expressions).
```

Comment:
Should this be pointing to the functions docs instead of expressions? https://www.packer.io/docs/from-1.5/functions

Also, I know you're saying we'll incorporate these docs ourselves for the final release. But for the beta it's probably a good idea at least acknowledge the docs we're linking are for Packer so that it's not confusing.

Reply:
Agreed - I intended to copy functions indeed :( .